packages/wrapper/src/editorAppClassic.ts

```diff
@@ -44,8 +45,17 @@ export class EditorAppClassic extends EditorAppBase {
         };
     }
 
+    override async loadUserConfiguration() {
+        if (this.config.editorOptions?.['semanticHighlighting.enabled'] !== undefined) {
+            // use updateConfiguration here as otherwise semantic highlighting will not work
+            await updateUserConfiguration(JSON.stringify({
+                'editor.semanticHighlighting.enabled': this.config.editorOptions['semanticHighlighting.enabled']
+            }));
```

Comment:
Is it still required? as the default value is "configuredByTheme"

also, doing that will override any existing user configuration, better using `StandaloneServices.get(IConfigurationService).updateValue('editor.semanticHighlighting.enabled', true)` instead maybe

Reply:
Done ⬇️